src/main/java/org/janelia/saalfeldlab/n5/N5FSWriter.java

```diff
@@ -166,6 +166,16 @@ public boolean remove(final String pathName) throws IOException {
 		return !Files.exists(path);
 	}
 
+	@Override
+	public boolean deleteBlock(
+			final String pathName,
+			final long[] gridPosition) {
+		return Paths
+				.get(basePath, getDataBlockPath(pathName, gridPosition).toString())
+				.toFile()
+				.delete();
```

Comment:
Access to block files is protected by file locking in other places. We could do the following to be consistent with it:
```java
try (final LockedFileChannel channel = LockedFileChannel.openForWriting(path)) {
	Files.delete(path);
}
```
Also, `Files.delete()` would throw an `IOException` in case of failure, and I would prefer this here because it's useful for diagnostics. The only problem may be that it would throw `NoSuchFileException` if the file doesn't exist, and we probably don't want it, since it was decided against `blockExists()` so there is no way of knowing beforehand.

How about returning `false` if the block file didn't exist, otherwise throwing an exception if it existed but the deletion failed?

Reply:
Good point.

Reply:
Just realized that what I suggested has a problem: if the file doesn't exist, the file channel will create it, and then it will be immediately deleted by `deleteIfExists`. So the file existence check would probably need to be performed before the lock

Reply:
I updated my PR with this but I had to add an additional `if (!Files.exist(path)) return false;` check to run the tests successfully, for some reason.